marklogic-client-api/src/main/java/com/marklogic/client/dataservices/impl/CallBatcherImpl.java

```diff
@@ -455,13 +460,24 @@ CallArgsImpl addDefaultArgs(CallArgsImpl callArgsImpl) {
 		if(defaultArgs == null || defaultArgs.size()==0)
 			return callArgsImpl;
 		CallerImpl callerImpl = (com.marklogic.client.dataservices.impl.CallManagerImpl.CallerImpl) caller;
-		List<CallField> newCallFields = new ArrayList<CallField>();
+		Set<CallField> newCallFields = new HashSet<CallField>();
```

Comment:
Instead of converting newCallFields to a List, does it make more sense to create it as a list originally?

Would that make line 479 unnecessary?